src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,88 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import classNames from 'classnames';
+import { useAppSelector } from '../../hooks/hooks';
+import { useDispatch } from 'react-redux';
+import { currentTodoSlice } from '../../features/currentTodo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { loaderSlice } from '../../features/loader';
 
 export const TodoModal: React.FC = () => {
+  const todo = useAppSelector(state => state.currentTodo);
+  const [user, setUser] = useState<User | null>();
```

Comment:
The `useState` for `user` should be initialized with `null` instead of leaving it undefined. This ensures that the state is explicitly set to `null` initially, avoiding potential issues with undefined values.